src/datastore/local-store/repos/api-key.repo.spec.ts

```diff
@@ -5,7 +5,7 @@ import { ApiKeyRepo } from '~/auth/repos/api-key.repo';
 import { LocalApiKeysRepo } from '~/datastore/local-store/repos/api-key.repo';
 import { defaultUser } from '~/users/user.consts';
 
-describe(`LocalApiKeyRepo does not meet ${ApiKeyRepo.type} test suite requirements`, () => {
```

Comment:
I updated the spec description here, since it was written for the failed report message, but looked wrong in review. The change is only cosmetic here from the `describe` string update